ui_components/src/travel/ScoutArea.js

```diff
@@ -64,7 +66,7 @@ const Player = ({
             </div>
             <div className='travel-scout-attack'>
                 {(player_data.attack === true && parseInt(player_data.battle_id, 10) === 0) && (
-                    <a href={attackLink + '&attack=' + player_data.attack_id}></a>
+                    <a onClick={() => attackLink(player_data.attack_id)}></a>
```

Comment:
Huh, if this is now not a string with the attack link but is a function that conducts the attack, could we change the name to reflect that? Maybe
```suggestion
                    <a onClick={() => attackPlayer(player_data.attack_id)}></a>
```

Reply:
Fixed in commit.